setup.py

```diff
@@ -61,6 +61,7 @@ def run_tests(self):
         "Programming Language :: Python :: 3.5",
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
```

Comment:
This should also be added to [tox.ini](https://github.com/backbonesk/cheddar-python/blob/master/tox.ini) to make sure we run the two tests on it.